biz.gemspec

```diff
@@ -15,6 +15,5 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.3'
 
-  gem.add_dependency 'clavius', '~> 1.0'
```

Comment:
Now that `v1.0.4` of `clavius` is released, let's go back to the same dependency configuration. If someone is looking for JRuby compatibility, they should upgrade to the latest `biz` and `clavius`.